oom-frontend/src/app/shared/modules/user/user-finder/user-finder.component.ts

```diff
@@ -0,0 +1,54 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {UsersBackendService} from "../../../../core/users/users-backend.service";
+import {User} from "../../../models/user.model";
+
+@Component({
+  selector: 'user-finder',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533382)
```suggestion
  selector: "user-finder",
```